### PR TITLE
cilium/cmd: Cosmetic code clean-ups

### PR DESCRIPTION
**Pull request by twpayne**
This PR makes the code a little more consistent in terms of formatting and structure. There is no functional change.

### CODE REVIEW

**Review by tklauser — APPROVED** [review]
Nice

**Comment by tklauser** [comment]
test-me-please

**Comment by tklauser** [comment]
Given that these are cosmetic changes, I've marked this PR to be backported for 1.9 in order to avoid merge conflicts in potential future 1.9 backports.

File: cilium/cmd/monitor.go [hunk 1]
```diff
@@ -196,11 +195,7 @@ func endpointsExist(endpoints format.Uint16Flags, existingEndpoints []*models.En
 		}
 	}
 
-	if len(endpointsFound) == 0 {
-		return false
-	}
-
-	return true
+	return len(endpointsFound) > 0
```
Comment (sayboras) [inline cilium/cmd/monitor.go:198]: nice, i think gosimple will flag similar cases as well. (e.g. golangci-lint run -E gosimple ./...)

**Review by sayboras — APPROVED** [review]
LGTM :100:

**Comment by tklauser** [comment]
retest-gke

**Review by christarazi — APPROVED** [review]

**Comment by pchaigno** [comment]
All team reviews have been addressed and CI is passing. Merging.

**Pull request merged**